### PR TITLE
MAINT: Better issue forms

### PR DESCRIPTION
**Pull request by larsoner**
Follows https://github.com/mne-tools/mne-bids/pull/1056 but adapts to MNE-Python

@drammock feel free to look and merge if you're happy

### CODE REVIEW

**Review by hoechenberger — APPROVED** [review]
Nice!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+labels: ["BUG"]
+
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Please use the [MNE-Python forum](https://mne.discourse.group/) to ask (and answer)
+        concrete technical questions or about usage of MNE-BIDS.
+        Use this form if you have experienced bugs.
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]:
```suggestion
        concrete technical questions or about usage of MNE-Python.
        Use this form *only* if you think there is a bug in MNE-Python code.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      label: Actual results
+      description: |
+        Please provide the actual output, traceback, screenshot, or another description of the results.
+
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: |
+        Replace this text with information about your system. For example through
+        using MNE-Python and running `mne.sys_info()` and pasting the output here.
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/bug_report.yml:50]:
```suggestion
        Tell us about your system: paste the output of `mne.sys_info()` here.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  - type: textarea
+    attributes:
+      label: Description of the problem
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Replace this text with a code snippet or minimal working example
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/bug_report.yml:27]:
```suggestion
        Paste here a code snippet or minimal working example
```

Comment (drammock) [reply .github/ISSUE_TEMPLATE/bug_report.yml:27]: Is there a way to make this field automatically formatted as python code? If so, maybe we should make a separate field for providing a URL for non-built-in datasets.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name: 📄 Documentation
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Request a new tutorial, an improvement to an existing tutorial, or a new term in our glossary.
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/documentation.yml:6]:
```suggestion
description: Tell us about tutorials you wish we had in our docs, ideas for improving existing tutorials, terms missing from our glossary, or parts of our docstrings that are confusing or inaccurate.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+labels: ["ENH"]
+
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Please use the [MNE-Python forum](https://mne.discourse.group/) to ask (and answer)
+        concrete technical questions or about usage of MNE-BIDS.
+        Use this form if you want to suggest new features.
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/feature_request.yml:17]:
```suggestion
        concrete technical questions or about usage of MNE-Python.
        Use this form *only* if you want to suggest new features.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      description: |
+        Please provide a clear and concise description of what you want to add or change.
+
+  - type: textarea
+    attributes:
+      label: Describe your proposed implementation
+      description: |
+        Describe how you think the feature or improvement should be implemented (e.g.,
+        as a new method on an existing class? as new capability added to an existing
+        method?) If you're not sure, please delete this section and the next section.*
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/feature_request.yml:31]:
```suggestion
        method?)
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+name: 🛠️ Feature request
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Suggest an idea for this project
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/feature_request.yml:6]:
```suggestion
description: Tell us about algorithms, visualizations, or other functionality you wish MNE-Python included.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,50 @@
+name: 🐛 Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Create a report to help us improve
```
Comment (drammock) [inline .github/ISSUE_TEMPLATE/bug_report.yml:6]:
```suggestion
description: Tell us about errors in our computations or plots.
```

**Review by drammock — COMMENTED** [review]
Left some suggested wording improvements.  +1 for merge to try them out ASAP, we can always come back and tweak these again later.

**Review by hoechenberger — APPROVED** [review]

**Comment by larsoner** [comment]
@hoechenberger if you want to make similar changes for MNE-BIDS, feel free. The `validation` is cool, as is the `render`. Also the [feature request](https://github.com/mne-tools/mne-bids/blob/main/.github/ISSUE_TEMPLATE/feature_request.yml) boxes over there seem more appropriate to a bug report.

**Comment by larsoner** [comment]
@drammock I think I covered your comments, including the Python rendering. Feel free to merge if you're happy

**Review by drammock — APPROVED** [review]

**Comment by hoechenberger** [comment]
> @hoechenberger if you want to make similar changes for MNE-BIDS, feel free. The validation is cool, as is the render. Also the feature request boxes over there seem more appropriate to a bug report

Happily delegating to @sappelhoff 😁😁😁

**Pull request merged**